imap_processing/cdf/config/imap_lo_l1a_variable_attrs.yaml

```diff
@@ -436,62 +443,87 @@ num_completed:
   FIELDNAM: Number of spins completed
   LABLAXIS: number of spins completed
 
-acq_end:
+acq_start_sec:
+  <<: *default_uint32
+  DEPEND_0: epoch
+  VAR_TYPE: support_data
+  CATDESC: Acquisition Start
+  FIELDNAM: Acquisition Start Seconds
+  UNITS: ns
+  LABLAXIS: acquisition start seconds
```

Comment:
Inconsistent capitalization here. I guess lowercase "s" would probably be preferred? Also in other fields below this too.